cli/cmd/install-service-mirror.go

```diff
@@ -0,0 +1,138 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/linkerd/linkerd2/pkg/charts"
+	"github.com/linkerd/linkerd2/pkg/charts/servicemirror"
+	"github.com/linkerd/linkerd2/pkg/version"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"k8s.io/helm/pkg/chartutil"
+	"sigs.k8s.io/yaml"
+)
+
+type installServiceMirrorOptions struct {
+	namespace           string
+	controlPlaneVersion string
+	dockerRegistry      string
+	logLevel            string
+	uid                 int64
+	requeueLimit        int32
+}
+
+const helmServiceMirrorDefaultChartName = "linkerd2-service-mirror"
+
+func newCmdInstallServiceMirror() *cobra.Command {
+	options, err := newInstallServiceMirrorOptionsWithDefaults()
+	if err != nil {
+		fmt.Fprint(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	cmd := &cobra.Command{
+		Use:   "install-service-mirror [flags]",
+		Short: "Output Kubernetes configs to install Linkerd Service Mirror",
+		Long:  "Output Kubernetes configs to install Linkerd Service Mirror",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return renderServiceMirror(os.Stdout, options)
+		},
```

Comment:
I think we should probably make this command hidden for now until everything is landed end-to-end.

Reply:
This would be a good candidate for the `alpha` (etc) subcommand if we had that

Reply:
Made it hidden now.